tests/common.bash

```diff
@@ -368,7 +368,8 @@ function restart_crio_service() {
 # Configures containerd
 function overwrite_containerd_config() {
 	containerd_config="/etc/containerd/config.toml"
-	sudo rm -f "${containerd_config}"
+	base_config_dir=$(dirname "${containerd_config}")
+	sudo mkdir -p "${base_config_dir}"
```

Comment:
Hmm. I sincerely thought we were using `DirectoryOrCreate` type for the `hostPath` mount.

I prefer this approach, though, thanks @BbolroC!